search-app/src/cmr/search/services/query_service.clj

```diff
@@ -485,7 +485,7 @@
         pr-result-format (rfh/printable-result-format result-format)
         log-message (log-search-result-metadata hits "deleted collections"
                                                 total-took (:client-id context) (:token context) pr-result-format
-                                                "with params %s" (pr-str params))]
+                                                "" nil)]
     (info log-message)
```

Comment:
So for the "Found X granules/collections in Y ms from client Z logs", we also print out the params here.

Example)
Found 12 granules in 144 ms from client asf_search_v7.0.7, token_type null in format :umm-json-results 1.6.6 with params {...}

which can get quite large.

Since we already get the same params values further up the log chain from the "Searching for ...." logs that means these params logs in the "Found x granules..." logs are repeats and can be removed.

Open to thoughts

Reply:
I kept the params parameter in the log-search-result-metadata func intact, because another func calls it with "with JSON Query %s and query params...". I was close to removing this log altogether, but decided to keep the json query one.

Reply:
did someone not have a comment about passing 'nil' as a parameter before and not like the practice? Can't the function be changed to accept two fewer parameters?

Reply:
We're not going this route, so reverted the changes